README.md

```diff
@@ -32,6 +32,16 @@ jobs:
 
 The `key` param is optional and you can use it to invalidate cache.
 
+### Caching strategy
+
+Define how you want your cache to be used. `move` is the default strategy:
+
+| Name           | Description                                                               | Usage                      | Observations                                                                                           |
+| -------------- | ------------------------------------------------------------------------- | -------------------------- | ------------------------------------------------------------------------------------------------------ |
+| Move           | Moves cache on job start, moves back on job end                           | `strategy: move`           | Uses the filesystem move syscalls, cache saving and restoration is instant. Does not work across disks/remote shares. |
+| Copy           | Recursiverly copies cache on job start, refreshes cache on job end        | `strategy: copy`           | Works across disks/remote shares.                                                                                                       |
+| Copy Immutable | Recursiverly copies cache on job start, does not refresh cache on job end | `strategy: copy-immutable` | Works across disks/remote shares.                                                                                                       |
+
```

Comment:
I love this table! ❤️ 

My only complain is that it is a bit too tight. Can we maybe merge the `Name` and `Usage` columns to save some horizontal space?

Would also add some hints on when to use `copy` vs `copy-immutable`.

Reply:
@oNaiPs after the table, would also add a little sentence that points to `push.yml` as a source of usage examples.